scripts/version-updater.gradle

```diff
@@ -47,11 +65,70 @@ class UpdateCoreSdkVersion extends DefaultTask {
 
     @TaskAction
     void write() {
-        Properties gradleProperties = PropertiesIO.readProperties(getProject().gradlePropertiesFilePath)
-        gradleProperties[getProject().gliaCoreSdkVersionTagInProperties] = newCoreSdkVersion
-        PropertiesIO.saveProperties(gradleProperties, getProject().gradlePropertiesFilePath)
+        getProject().saveProperty(getProject().gliaCoreSdkVersionTagInProperties, newCoreSdkVersion)
+    }
+}
+
+/**
+ * A task class for editing the Widget SDK version in version.properties
+ * Example of usage from terminal:
+ * ./gradlew saveWidgetsVersionName --versionCode=77 --versionName=0.77.2
+ */
+class UpdateProjectVersion extends DefaultTask {
+    private String versionName
+    private String versionCode
+
+    @Option(option = "versionName", description = "Widget SDK version name that should be overriden in configuration files")
+    UpdateProjectVersion setVersionName(String versionName) {
+        this.versionName = versionName
+        return this
+    }
+
+    @Option(option = "versionCode", description = "Widget SDK version code that should be overriden in configuration files")
+    UpdateProjectVersion setVersionCode(String versionCode) {
+        this.versionCode = versionCode
+        return this
+    }
+
+    @TaskAction
+    void write() {
+        Map<String, String> propertiesMap = new HashMap<>();
+        propertiesMap.put(getProject().wdigetsVersionNameTagInProperties, versionName)
+        propertiesMap.put(getProject().wdigetsVersionCodeTagInProperties, versionCode)
+        getProject().saveProperties(propertiesMap)
     }
 }
 
 tasks.register('saveCoreSdkVersion', UpdateCoreSdkVersion)
+tasks.register('saveWidgetsVersionName', UpdateProjectVersion)
+
+/**
+ * This task is used by Bitrise release flow to dynamically increment the current Widgets SDK version
+ * Usage example:
+ * ./gradlew bumpVersionMinorLevel
+ */
+task bumpVersionMinorLevel {
+    doLast {
+        String[] versionPartitioned = widgetsVersionName.split("\\.")
+        int updatedMinorVersion = versionPartitioned[1].toInteger() + 1
+        versionPartitioned[1] = updatedMinorVersion.toString()
+        versionPartitioned[2] = "0"
+        String newVersionName = String.join(".", versionPartitioned)
+        String newVersionCode = (widgetsVersionCode + 1).toString()
+
+        Map<String, String> propertiesMap = new HashMap<>();
+        propertiesMap.put(getProject().wdigetsVersionNameTagInProperties, newVersionName)
+        propertiesMap.put(getProject().wdigetsVersionCodeTagInProperties, newVersionCode)
+        saveProperties(propertiesMap)
+    }
+}
+
+/**
+ * This task is used by Bitrise release flow to read current Widgets SDK version
+ * Usage example:
+ * ./gradlew bumpVersionMinorLevel
```

Comment:
I think here it's `./gradlew printCurrrentVersionName`

Reply:
Fixed: https://github.com/salemove/android-sdk-widgets/compare/adc0b3d4f881681adf33c2e97d9f03f228f53c35..88c6bcad845d0e3a0f3bb87ba914fa207e0ef505